fix google-sheets-spreadsheet-get command (#30170)
demisto/content: Packs/GoogleSheets/ReleaseNotes/1_0_34.md

#### Integrations
##### Google Sheets
- Updated the Docker image to: *demisto/googleapi-python3:1.0.0.77375*.

> - Fixed an issue which the ***google-sheets-spreadsheet-get*** command would fail when used with the *include_grid_data* argument and a value of `None` was present in grid-data.

Reviewer: ```suggestion
- Fixed an issue where ***google-sheets-spreadsheet-get*** failed when used with the *include_grid_data* argument, and a `None`-value was present in the grid data.
```
